tools/sggit/tools.go

```diff
@@ -26,7 +26,7 @@ func VerifyNoDiff(ctx context.Context) error {
 	if status.String() != "" {
 		dirtyDetails := status.String()
 		// attempt to give a nice patch output if the dirty files are tracked by git
-		if patchOutput := sg.Output(Command(ctx, "diff", "--patch")); patchOutput != "" {
+		if patchOutput := sg.Output(Command(ctx, "diff", "--name-only")); patchOutput != "" {
```

Comment:
This might break things in unexpected ways so let's not include in this PR :)

Reply:
Good idea, it was just something I have thought of.

Reply:
Fixed now.